src/sensors/RealSenseCamera.cc

```diff
@@ -41,6 +54,10 @@ RealSenseCamera::RealSenseCamera(size_t width, size_t height, unsigned int fps)
 
     this->scale = this->dev->get_depth_scale();
 
+    if (this->dev->supports_option(r200_opts[0])) {
+        this->dev->set_options(r200_opts, ARRAY_SIZE(r200_opts), r200_values);
```

Comment:
You could use a static local variable. So they are in the scope of the function. I prefer this way, but it is up to you.
Other than that, the patch looks ok for me.